frc53_nft/src/util.rs

```diff
@@ -28,14 +28,16 @@ impl OperatorSet for BitField {
     }
 }
 
-/// Maintains set-like invariants in-memory by maintaining sorted order of the underlying array
-/// Insertion and deletion are O(n) operations but we expect operator lists to be a relatively small size
-/// TODO: benchmark this against some other options such as
-/// - BTreeSet in memory, Vec serialized
-/// - BTreeSet in memory and serialization
-/// - HashSets...
-/// - Hamt<ActorID, ()>
-/// - Amt<ActorID>
+/// Maintains set-like invariants in-memory by maintaining sorted order of the underlying array.
+///
+/// Insertion and deletion are O(n) operations but we expect operator lists to be a relatively small
+/// size.
+// TODO: benchmark this against some other options such as
+// - BTreeSet in memory, Vec serialized
+// - BTreeSet in memory and serialization
+// - HashSets...
+// - Hamt<ActorID, ()>
+// - Amt<ActorID>
```

Comment:
this is going to work OK? `//` after the `///`? should you just stuck the TODO bit inside the `impl` maybe?

Reply:
I can move it inside, I just didn't want it in the doc comment.

Reply:
I moved it above to make it clear.